### PR TITLE
adds vault chart

### PR DESCRIPTION
**Pull request by xh3b4sd**
This will replace the old `vaultlab`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
-[![CircleCI](https://circleci.com/gh/giantswarm/e2esetup.svg?style=svg)](https://circleci.com/gh/giantswarm/e2esetup)
+[![CircleCI](https://circleci.com/gh/giantswarm/e2esetup.svg?style=shield)](https://circleci.com/gh/giantswarm/e2esetup)
```
Comment (xh3b4sd) [inline README.md:1]: Aligning with other projects.

File: helm/e2esetup-vault-chart/templates/vault.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
+apiVersion: apps/v1beta1
```
Comment (xh3b4sd) [inline helm/e2esetup-vault-chart/templates/vault.yaml:1]: This is all copied from https://github.com/giantswarm/vaultlab.

File: helm/e2esetup-vault-chart/values.yaml [hunk 3]
```diff
@@ -0,0 +1,2 @@
+vault:
+  token: "myToken"
```
Comment (xh3b4sd) [inline helm/e2esetup-vault-chart/values.yaml:2]: This here is different as in more structured.

**Review by kopiczko — APPROVED** [review]
Let's make sure there is corresponding app repo in quay.

**Comment by xh3b4sd** [comment]
I created it and it got already pushed into it.

**Pull request merged**